cmd/constants.go

```diff
@@ -64,7 +64,8 @@ const (
 
 // defaults
 const (
-	DEFAULT_GET_LIMIT = 10
+	DEFAULT_GET_LIMIT  = 100
```

Comment:
I'm cool with this - maybe we should change the default on the API to this as well?

Reply:
I thought about that as well.  Do you think any of the APIs will really return more than 100 results at a time?

Reply:
The snow max is absurdly high (I think it was 10k by default?) - so we're probably safe at 100 :smile:

Reply:
Yeah, 10k. :) Ok, I'm going to leave this then.